### PR TITLE
added an extra section instructing users to edit the initRegistry.js file

### PR DESCRIPTION
**Pull request by JosephBadua**

### CODE REVIEW

File: getting-started.md [hunk 1]
```diff
@@ -42,6 +42,17 @@ node createUser.js
 
  This will prompt you to set a username and password, and then create the entries in the mongo database. The password is hashed.
 
+## Edit the initRegistry.js file
+
+Inside initRegistry.js change the username and password to your own
+
+```bash
```
Comment (ankitskvmdam) [inline getting-started.md]: use Javascript instead of bash.

Comment (JosephBadua) [reply getting-started.md]: I have made the changes

Comment (ankitskvmdam) [reply getting-started.md]: Perfect!

Comment (JosephBadua) [reply getting-started.md]: Thank you @ankitskvmdam  for the opportunity, this has been my first open-source contribution.

**Pull request merged**